### PR TITLE
Luigi

### PR DESCRIPTION
**Pull request by Godox**
Because Luigi is better than Mario !

### CODE REVIEW

**Comment by Godox** [comment]
And gradle better than nothing

**Comment by KikiManjaro** [comment]
Nice ! Could you add some screenshot to the README as well ?

**Comment by Godox** [comment]
Done :)

File: src/main/java/manjaro/mpb/config/MarioProgressBarSettingsComponent.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+public class MarioProgressBarSettingsComponent {
+
+    private final JPanel configMainPanel;
+
+    private final List<JBRadioButton> charactersRadioButtons = new ArrayList<>();
+
+    public MarioProgressBarSettingsComponent() {
+        JBLabel title = new JBLabel("Chose your character :", UIUtil.ComponentStyle.REGULAR);
```
Comment (KikiManjaro) [inline src/main/java/manjaro/mpb/config/MarioProgressBarSettingsComponent.java:25]:
```suggestion
        JBLabel title = new JBLabel("Choose your character :", UIUtil.ComponentStyle.REGULAR);
```

File: README.md [hunk 2]
```diff
@@ -7,12 +7,21 @@ The Mario progress bar for IntelliJ IDEA and other JetBrains IDEs.
...
-
 This is the [Mario Progress Bar](https://plugins.jetbrains.com/plugin/14708-mario-progress-bar/versions) for [JetBrains IntelliJ IDEA](https://www.jetbrains.com/idea/) based on the Nyan Progress Bar by Dmitry Batkovitch. It display Mario with some bricks for the basic bar and a Koopa Shell for the indeterminate one.
 
+## Configuration
+
+You can access plugin's configuration in :
+
+Settings > Tools > Mario Progress Bar 
+
+Here's you can select your character.
```
Comment (KikiManjaro) [inline README.md:22]:
```suggestion
You can choose your character here.
```

**Pull request merged**